api/config/module/feedback/v1/feedback.proto

```diff
@@ -10,7 +10,7 @@ import "feedback/v1/feedback.proto";
 message Survey {
   string prompt = 1 [ (validate.rules).string = {min_bytes : 1} ];
   string freeform_prompt = 2;
-  clutch.feedback.v1.RatingLabels rating_labels = 3 [ (validate.rules).message = {required : true} ];
+  repeated clutch.feedback.v1.RatingLabel rating_labels = 3 [ (validate.rules).repeated = {min_items : 2} ];
```

Comment:
currently we have a 3-point scale but keeping this general to future cases where we support additional scales. but i think we can assume that the rating scale should at least have 2 items.